js/repeater-list.js

```diff
@@ -379,7 +379,7 @@
 			this.viewOptions.list_rowRendered({
 				container: $tbody,
 				item: $row,
-				rowData: data
+				rowData: data.items[index]
```

Comment:
Is there any case where it isn't just a convenience accessor for the data.items param?